src/utils/envVars.ts

```diff
@@ -7,7 +7,10 @@ export const ENABLE_RPC_FEATURES        = Boolean(INFURA_PROJECT_ID && INFURA_PR
 export const INFURA_URL                 = `https://${IS_MAINNET ? "mainnet" : "goerli"}.infura.io/v3/${INFURA_PROJECT_ID}`;
 
 // public
-export const ETH2_NETWORK_NAME          = process.env.REACT_APP_ETH2_NETWORK_NAME   || 'Mainnet';
+export const TESTNET_LAUNCHPAD_NAME      = 'Pyrmont'
```

Comment:
I think this should still be configurable via a `process.env.` env-var as there are times when we have multiple concurrent testnets and ideally, the launchpad is not where this is configured. You shouldn't have to change the source code to change the testnet name.